packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -402,3 +407,21 @@ const enableTooltips = async (page: Page): Promise<void> => {
 const hideVideoLoadingSpinner = async (page: Page): Promise<void> => {
   await page.addStyleTag({ content: '[data-ui-id="stream-media-loading-spinner"] {display: none}' });
 };
+
+/**
+ * Helper function for waiting for file type icons.
+ */
+const awaitFileTypeIcon = async (page: Page): Promise<void> => {
+  const fileTypeIconId: string = dataUiId(IDS.fileTypeIcon);
+  await waitForFunction(
+    page,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    (args: any) => {
+      const iconNodes = Array.from(document.querySelectorAll(args.fileTypeIconId));
+      return iconNodes.every((node) => node?.querySelector('img').height > 0);
+    },
+    {
+      fileTypeIconId: fileTypeIconId
+    }
+  );
+};
```

Comment:
This PR looks good to me. Just wondering if we should go one step further and query all icon tags and wait for all height is > 0 in the `stableScreenshot` function. Thoughts @JamesBurnside @dmceachernmsft? I know we register other icons so they load faster but the file type icon is determined dynamically (using function `getFileTypeIconProps` from '@fluentui/react-file-type-icons') and can't be registered.

Reply:
this to me seems tricky, what if a Icon isn't in a snapshot but we are waiting for it in the stable screenshot method? seems to me that this might be the way to do it with a series of helper functions when we run into a icon like this we can't register?

Reply:
It should be fine for snapshots without icons since we can return true if iconNodes is empty after running:
`const iconNodes = Array.from(document.querySelectorAll('i');`
But I'll just resolve this thought as it is outside the scope of this ticket. I'll just keep this in mind the next time we fail UI tests because icons don't load in time.

Reply:
Another option would be to mock the `@fluentui/react-file-type-icons` package at build time.. but don't love that...